dev/tasks/tasks.yml

```diff
@@ -416,7 +416,7 @@ tasks:
 
 {############################## Wheel macOS ####################################}
 
-{% for macos_version, macos_codename in [("13.0", "ventura")] %}
+{% for macos_version, macos_codename in [("12.0", "monterey")] %}
```

Comment:
Not that important, but we could also do this without a for loop given there is only one entry, like is done for the arm64 wheel below

Reply:
Fair! Done!